repair/featurize/freqfeat.py

```diff
@@ -12,24 +12,25 @@ def specific_setup(self):
         total, single_stats, pair_stats = self.ds.get_statistics()
         self.total = total
         self.single_stats = single_stats
+        # The query results used to featurize the dataset
```

Comment:
Can you add a comment here denoting the structure of `featurization_query_results` e.g.
```
List of tuples (vid, attribute, ||| separated domain values)
```